content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -77,12 +77,19 @@ EndpointSlices support three address types:
 
 * IPv4
 * IPv6
-* FQDN (Fully Qualified Domain Name)
+* FQDN (Fully Qualified Domain Name) [DEPRECATED]
 
 Each `EndpointSlice` object represents a specific IP address type. If you have
 a Service that is available via IPv4 and IPv6, there will be at least two
 `EndpointSlice` objects (one for IPv4, and one for IPv6).
 
+{{< note >}}
+
+EndpointSlice.AddressType value "FQDN" is deprecated. For more details, you can read the
+[Generate warning for EndpointSlice AddressType FQDN](https://github.com/kubernetes/kubernetes/pull/114677)
```

Comment:
I don't think it is a good idea to link to the upstream source code change.
The documentation should focus on the intent and the reason behind this deprecation,
rather than pointing to the source code change.
I'd suggest we put a **very brief** summary for the discussion [here](https://github.com/kubernetes/kubernetes/issues/114210).
Even if a simple statement like "FQDN was deprecated due to lack of clearly defined semantics" would be better than pointing users to the code change.

Reply:
Done.